src/widgetastic/widget/table.py

```diff
@@ -370,6 +370,7 @@ class MyCustomTable(Table):
         bottom_ignore_fill: Whether to also strip these top rows for fill.
     """
     ROWS = './tbody/tr[./td]|./tr[not(./th) and ./td]'
+    EMPTY_CONTENT = './/tbody[.//*[contains(@class,"ins-c-table__empty")]]'
```

Comment:
Why adding a UI library-specific check into the standard library? I can imagine that such check can be added as an optional way to help the table decide but it already is pretty bloated with functionality :)

Reply:
@Victoremepunto please submit your changes in widgetastic.patternfly4 library https://github.com/RedHatQE/widgetastic.patternfly4/blob/master/src/widgetastic_patternfly4/table.py